src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultSecretAttribute.md

```diff
@@ -1,4 +1,4 @@
-﻿---
```

Comment:
@maddieclayton same comment about this call being a [`PATCH`](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault/Generated/KeyVaultClient.cs#L4792), so we should change the verb to `Update`

Reply:
Going to rename to Update-AzureKeyVaultSecret as I will combine this with Set-AzureKeyVaultSecret at the breaking change release and I don't want to aliases to get out of control.